src/xpk/core/core.py

```diff
@@ -1675,7 +1675,8 @@ def build_docker_image_from_base_image(args, verbose=True) -> tuple[int, str]:
   )
   tmp = write_tmp_file(docker_file)
   docker_build_command = (
-      f'docker build -f {str(tmp.file.name)} -t {docker_name} {args.script_dir}'
+      f'docker buildx build --platform=linux/amd64 -f {str(tmp.file.name)} -t'
```

Comment:
Please move value of platform to variable.

Reply:
Can we have linux/arm64 set here?

Reply:
> Can we have linux/arm64 set here?

Should we build for both platforms, or should we add a flag to allow the user to configure this? WDYT?

Reply:
I think we may add flag to allow user to configure it, with default value set to amd64. Please test how it works with this flag set to linux/arm64, if there are any problems than we should remove it.

Reply:
It's not working with current configuration. I think we also need to have node pool with linux/arm64 to run it.

Reply:
ok, than leave it as it is